### DISCUSSION TITLE
Hi when i am trying to get Domain name Hostinger site requesting for auto pay for 10k. please let me know whether it will deduct Rs10k or Rs89+gst = Rs122 and here i am attaching the screen shot

### DISCUSSION THREAD

**Discussion by shouryagoud** [learndevopsonline/learndevopsonline#1573] [Q&A]
![Screenshot_2023-11-07-11-35-23-64_944a2809ea1b4cda6ef12d1db9048ed3](https://github.com/learndevopsonline/learndevopsonline/assets/114072857/a715547d-7ab2-4e38-a8cc-0b0d1d4a9de9)

**Comment by sreedharm07** [comment]
can you send the hostinger screenshot before you giving payment

**Comment by shouryagoud** [comment]
![IMG20231107112739](https://github.com/learndevopsonline/learndevopsonline/assets/114072857/df2c3a81-4933-4f4b-84a4-ff42e5ec5a0c)

**Reply by sreedharm07** [reply]
you are giving autopay dont use autopay

**Reply by shouryagoud** [reply]
actually i didn't give any auto pay. in hostinger page after click on continue it will directly taking to payment option which i attached previously. and here my doubt is if i continue with the payment option, will it deduct the Rs10k or Rs122 and shall i continue with the payment option or not

**Reply by sreedharm07** [reply]
actually it will take 122 only and after payment we should cancel autopay, autopay is for after your domain expire it will pay automaticallyfor your domain but if it deduct 10k we cont get it back so,
change your payment method and try with credit or debit card. 
if you use card you will get otp with accurate amount that is going to deduct.

**Reply by shouryagoud** [reply]
Okay .i will try

**Reply by shouryagoud** [reply]
Paid via card. it's taken 122. Done. Thanq
